sympy/polys/rings.py

```diff
@@ -1405,7 +1449,7 @@ def quo_ground(self, x):
             return self
         return self._quo_ground(x)
 
-    def extract_ground(self, g):
+    def extract_ground(self, g: PolyElement[Er]) -> tuple[Er | PolyElement[Er], PolyElement[Er], PolyElement[Er]]:
```

Comment:
The first item in the return type should just be `Er` not `Er | PolyElement[Er]`.